### PR TITLE
Issue template updates

### PR DESCRIPTION
**Pull request by IsaacMarovitz**

### CODE REVIEW

**Review by Depal1 — CHANGES_REQUESTED** [review]
The game support area could be improved further to make reviews easier.
`Did the app install correctly? -> Does it load without crashing? -> Does it crash in-game with a known trigger?`

File: .github/ISSUE_TEMPLATE/FEATURE-REQUEST.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+name: Feature Request
+description: Request a new feature
+title: "[FR]: "
```
Comment (Candygoblen123) [inline .github/ISSUE_TEMPLATE/FEATURE-REQUEST.yml]: can you make this `[Feature]: ` or `[Feat]: `

File: .github/ISSUE_TEMPLATE/GAME-SUPPORT.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+name: Game Support Request
+description: Request a support for a currently unsupported game
+title: "[GS]: "
```
Comment (Candygoblen123) [inline .github/ISSUE_TEMPLATE/GAME-SUPPORT.yml]: can you change this to `[Game Support]: `

**Review by Candygoblen123 — CHANGES_REQUESTED** [review]

**Review by Depal1 — APPROVED** [review]

**Review by Candygoblen123 — APPROVED** [review]

**Review by danielrominger46 — COMMENTED** [review]
Testes

**Pull request merged**